scripts/kubectl-kubestellar-ensure-wmw

```diff
@@ -17,10 +17,12 @@
 # Purpose: ensure that a workload management workspace exists and has
 # the needed APIBindings.
 
-# Usage: kubectl ws parent; $0 ($kubectl_flag | --with-kube boolean)* wmw_name
+# Usage: $0 ($kubectl_flag | --with-kube boolean)* wmw_path
```

Comment:
This script is documented.  You have to change the documentation along with the behavior of the script.

Reply:
Can you give me a pointer to this document?